app/packages/state/src/recoil/schema.ts

```diff
@@ -356,7 +356,7 @@ export const labelPath = selectorFamily<string, string>({
 
 export const _activeFields = atomFamily<string[], { modal: boolean }>({
   key: "_activeFields",
-  default: null,
+  default: [],
```

Comment:
This was tricky to track down - the `selectorFamily` below returns `labelFields()` when this variable is null